avocado/utils/iso9660.py

```diff
@@ -160,7 +166,30 @@ def close(self):
         pass
 
 
-class Iso9660IsoInfo(BaseIso9660):
+class BaseIso9660MntDirMount(BaseIso9660):
```

Comment:
can you please expand you thoughts? I'm not sure what do you mean by mix in class and what do you propose?